src/blockchain/alt_block_tree.cpp

```diff
@@ -76,7 +76,7 @@ bool payloadsCheckDuplicates(Index& index,
 template <typename Index, typename Pop>
 void commitPayloadsIds(Index& index, const std::vector<Pop>& pop) {
   for (const auto& p : pop) {
-    index.template insertPayloadIds<Pop, typename Pop::id_t>(p.getId());
+    index.template insertPayloadId<Pop>(p.getId());
   }
```

Comment:
maybe add method `insertPayloadsIds<Pop>(pop)`? inserting vector into another is more effective than just inserting same vector 1by1